src/platform/audio/ffmpegaudiofilecache.cpp

```diff
@@ -337,7 +349,7 @@ void *FFmpegAudioFileCache::Open_File(AudioEventRTS *audio_event)
     }
 
     // Load the file from disk
-    File *file = g_theFileSystem->Open(filename, File::READ | File::BINARY | File::BUFFERED);
+    File *file = g_theFileSystem->Open(filename, File::READ | File::BINARY);
```

Comment:
`File::BUFFERED` was removed again. Was there something wrong with it?

Reply:
No, it worked fine. I was trying to figure out an error and `MilesAudioFileCache` doesn't have this